setup.py

```diff
@@ -5,7 +5,7 @@
 
 setup(
     name='amundsen-common',
-    version='0.3.6',
+    version='0.4.6',
```

Comment:
If you're bumping the minor version then the patch version restarts at 0, so `0.4.0`